src/collection/collection_test.go

```diff
@@ -119,6 +122,7 @@ func TestBuildCollectionList_DetailedList(t *testing.T) {
                           "ignored_database": {"schema1": { "table1": ["index1"] }}
                          }`,
 		CollectionIgnoreDatabaseList: `["ignored_database"]`,
+		CollectionIgnoreTableList:    `["ignored_table"]`,
```

Comment:
Is this a meaningful test?
I was expecting a test collecting everything "without that options" and a second test collecting everything but the specified tables "with that option enabled"

Reply:
I have added a new test for this new table configuration and removed the configuration in the other tests where it did not make sense. Commit [#1db6e3f](https://github.com/newrelic/nri-postgresql/pull/164/commits/1db6e3f81300cf5bfca8d375b5d31d3cd4f02b14)